CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Support for using Apple Accelerate as the BLAS library
 
 ### Fixed
+- Fix building server with Boost 1.75
```

Comment:
Please move it to a new `### Fixed` section under the `## [Unreleased]`.

Reply:
Done